### ISSUE TITLE
questions about point_dim

### ISSUE THREAD

**Issue by xz-123-new** [ShunChengWu/3DSSG#38]
In the the scene graph prediction task with ground truth segmentation mask, only xyz coordinates of point cloud is utilized, i'm curious that is there some limitation of this problem setting that other features like rgb information and normal vectors are forbidden to use? Or just because they are not crucial for this problem? Thanks !

**Comment by ShunChengWu** [comment]
We did an ablation study on those two extra inputs. They provide some improvement (especially normal) but are very minor. 
This repo extracts rgb information in runtime, which slows down the training significantly. If you would like to use them, you can write a new dataloader.
